Library/Formula/unarj.rb

```diff
@@ -0,0 +1,15 @@
+class Unarj < Formula
+  homepage "http://www.arjsoftware.com/files.htm"
+  url "http://pkgs.fedoraproject.org/repo/pkgs/unarj/unarj-2.65.tar.gz/c6fe45db1741f97155c7def322aa74aa/unarj-2.65.tar.gz"
+  sha1 "e30242ab0cb521bae89472a552219a06fdddb389"
+  version "2.65"
+
+  def install
+    system "make" 
+    bin.install "unarj"
+  end
+
+  test do
+    system "unarj"
```

Comment:
Sure! You could add one of those as a `resource` in the formula, and stage it during the test.